dpnp/fft/dpnp_utils_fft.py

```diff
@@ -119,7 +174,7 @@ def _compute_result(dsc, a, out, forward, c2c, a_strides):
     else:
         if (
             out is not None
-            and out.strides == a_strides
+            and out.strides == out_strides
             and not ti._array_overlap(a_usm, dpnp.get_usm_ndarray(out))
         ):
             res_usm = dpnp.get_usm_ndarray(out)
```

Comment:
please add a test to cover that

Reply:
The issue was in the fact that `out_strides` in now a list compared with `out.strides` which is a tuple. The code itself is already covered with previous tests.